src/cmd-buildextend-metal

```diff
@@ -287,18 +274,8 @@ if [ "${OSBUILD_SUPPORTED:-}" != "" ] && [ "${COSA_USE_OSBUILD:-}" != "0" ]; the
                 --mpp "/usr/lib/coreos-assembler/osbuild-manifests/coreos.osbuild.${basearch}.mpp.yaml" \
                 --filepath "${path}.tmp" "${osbuild_extra_args[@]}"
 else
-    runvm "${qemu_args[@]}" -- \
-            /usr/lib/coreos-assembler/create_disk.sh \
-                --config "${image_for_disk_json}" \
-                --platform "${ignition_platform_id}" \
-                --platforms-json "${platforms_json}" \
-                "${disk_args[@]}"
-
-    # Now that we've created the image let's resize it to the desired
-    # size for qemu (and all images based on it).
-    if [[ $image_type == qemu ]]; then
-        qemu-img resize "${path}.tmp" "${cloud_image_size_mb}M"
-    fi
+    echo "OSBUILD_SUPPORTED is not supported or COSA_USE_OSBUILD is unset"
+    exit 1
```

Comment:
I think let's just simplify this further and not have any conditionals checking env vars anymore.